personalities/sctfe/storage.go

```diff
@@ -40,6 +41,10 @@ type Storage interface {
 	Add(context.Context, *ctonly.Entry) (uint64, error)
 	// AddIssuerChain stores every the chain certificate in a content-addressable store under their sha256 hash.
 	AddIssuerChain(context.Context, []*x509.Certificate) error
+	// AddCertIndex stores the index of certificate in a content-addressable store.
```

Comment:
nit: these aren't stored in a CAS, or the address would be derived from the index rather than the cert.

Reply:
fair enough!